### PR TITLE
Better MusicBrainz Lookups Messages

### LINKED ISSUE #220: [Enhancement] Different message when fetching metadata from MusicBrainz fails
Currently, when I try to fetch metadata using MusicBrainz for a given file the following message appears "Downloaded metadata for 0 files successfully" and no tag is changed.

I can find the songs on MusicBrainz but I'm not sure how to compare the AccoustId fingerprints (available as an image in the website, I think?) with the one provided in Tagger's interface. However, I'm almost sure they don't match, since my files usually have a different length from the ones provided AccoustId. I don't know if I can actually upload the music file here, but I tested with "George Ezra's Budapest" youtube version, which has a length of 3m37s versus the range 3m24-3m28 in [AccoustId](https://musicbrainz.org/recording/c63c4b83-2144-4c93-a6d4-39e85278d809/fingerprints).

Considering this, I think that the fetching itself fails since there is no match, but I can't be sure because the message is uninformative. In such cases I think a different message would be better.

## System information

Tagger version: 2023.8.1 (from flathub)
OS version: Ubuntu 23.04
Session type: wayland

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #220 

Screenshot from 2023-08-04 22-29-41

image

### CODE REVIEW

**Comment by fsobolev** [comment]
Since the list of files in dialog can be pretty long, I think it worth trying to use the style of dialog we use for example for corrupted files, so there will be scrollable view.

**Comment by nlogozzo** [comment]
> Since the list of files in dialog can be pretty long, I think it worth trying to use the style of dialog we use for example for corrupted files, so there will be scrollable view.

Ok, will implement that :)

**Comment by nlogozzo** [comment]
image

File: NickvisionTagger.GNOME/Blueprints/scrolling_message_dialog.blp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        orientation: vertical;
+
+        Gtk.Label _titleLabel {
+          styles ["title-2"]
+          margin-bottom: 24;
+        }
+
+        Gtk.Separator {}
+      
+        Gtk.ScrolledWindow _scrolledWindow {
```
Comment (fsobolev) [inline NickvisionTagger.GNOME/Blueprints/scrolling_message_dialog.blp:29]: `hscrollbar-policy: never`

**Review by fsobolev — CHANGES_REQUESTED** [review]

**Comment by nlogozzo** [comment]
image

**Comment by fsobolev** [comment]
Hm... Shouldn't the test wrap? Maybe we forgot to set some property...

**Comment by nlogozzo** [comment]
> Hm... Shouldn't the test wrap? Maybe we forgot to set some property...

I set `wrap-mode: word;` but it wasn't working correctly. It put everything on one line and because hscroll was disabled and just wouldn't show it.

Let me try `wrap-mode: char;`

**Comment by nlogozzo** [comment]
Ok I set `wrap-mode` in latest commit, but it still seems to be a long dialog....

File: NickvisionTagger.Shared/Models/MusicFile.cs [hunk 2]
```diff
@@ -304,15 +315,19 @@ public async Task<bool> LoadTagFromMusicBrainzAsync(string acoustIdClientKey, st
             var bestResult = response.Results[0];
             foreach (var r in response.Results)
             {
-                if(r.Score > bestResult.Score && r.Recordings.Count > 0)
+                if (bestResult.Recordings.Count == 0 && r.Recordings.Count > 0)
```
Comment (DaPigGuy) [inline NickvisionTagger.Shared/Models/MusicFile.cs:318]: `(r.Score > bestResult.Score || bestResult.Recordings.Count == 0) && r.Recordings.Count > 0`

File: NickvisionTagger.GNOME/Controls/ScrollingMessageDialog.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+using NickvisionTagger.GNOME.Helpers;
+
+namespace NickvisionTagger.GNOME.Controls;
+
+public partial class ScrollingMessageDialog : Adw.Window
+{
+    [Gtk.Connect] private readonly Gtk.Label _titleLabel;
+    [Gtk.Connect] private readonly Gtk.ScrolledWindow _scrolledWindow;
```
Comment (DaPigGuy) [inline NickvisionTagger.GNOME/Controls/ScrollingMessageDialog.cs:8]: Unused

**Review by DaPigGuy — CHANGES_REQUESTED** [review]

**Pull request merged**